Car/TeamNull SmartCar/src/util/HardwareUtil.cpp

```diff
@@ -121,13 +191,16 @@ int getFrontDistance() {
 }
 
 int getLeftFrontDistance() {
-  return 100;
+  return frontLeft.getMedianDistance(2);
```

Comment:
What's the point of getting the median distance of two measurements? You'd need at least three for a median to make sense.

Reply:
We had some issues with code getting stuck 'somewhere' and so this was lowered from 5 to 2 for testing, but yes, it is intended to be higher